content/applications/finance/accounting/taxes/vat_validation.rst

```diff
@@ -1,43 +1,44 @@
-===========================
-VIES VAT numbers validation
-===========================
+===============================
+VAT numbers verification (VIES)
+===============================
```

Comment:
Ah, yes I did. My thought process was that the page title should match the name of the feature in Odoo, but then I realized that it would make it easier to search for if the title included VIES. We can keep the original title.

Reply:
I didn't realize, last Friday, that the different titles use different words "validation" and "verification".
I agree it's best to follow the feature's name. It's probably because this page talked about VIES verification and VAT number formatting validation.